source/compiler/dttemplate.h

```diff
@@ -1896,7 +1896,7 @@ const unsigned char TemplateSlit[] =
 
 const unsigned char TemplateSpcr[] =
 {
-    0x53,0x50,0x43,0x52,0x50,0x00,0x00,0x00,  /* 00000000    "SPCRP..." */
+    0x53,0x50,0x43,0x52,0x5A,0x00,0x00,0x00,  /* 00000000    "SPCRZ..." */
     0x01,0xE3,0x49,0x4E,0x54,0x45,0x4C,0x20,  /* 00000008    "..INTEL " */
```

Comment:
I think you need to update the revision and checksum as well.

Reply:
noted. will do

Reply:
Thanks @vlsunil for the review, I will take a look as well next week once those suggestions have been taken, thanks!

Reply:
Please don't forget to update the template too once all the changes are made!

Reply:
Hi @xiaobo55x , @vlsunil , @sacdintel ,

I have updated the code. Please help to review again. Thanks.